app/pages/resource/reservation-calendar/ReservationCalendarContainer.js

```diff
@@ -80,7 +80,11 @@ export class UnconnectedReservationCalendarContainer extends Component {
     const endSlot = last(orderedSelected);
     const beginText = this.getDateTimeText(beginSlot.begin, true);
     const endText = this.getDateTimeText(endSlot.end, false);
-    return `${beginText} - ${endText}`;
+    const duration = moment.duration(moment(endSlot.end).diff(moment(beginSlot.begin)));
+    const hours = duration.hours();
+    const mins = duration.minutes();
+    const durationText = `${hours > 0 ? `${hours}h ` : ''}${mins}min`;
+    return `${beginText} - ${endText} (${durationText})`;
```

Comment:
Would it be better if we can just use moment.format with specific format here?

Reply:
`moment.format` is not flexible enough to handle this case.